models/app.models.js

```diff
@@ -68,4 +68,32 @@ exports.fetchCommentsByArticleId = async id => {
 	} else {
 		return results[1].rows;
 	}
+};
+
+exports.createNewComment = async (id, username, body) => {
+	const articleQuery = db.query(
```

Comment:
You are likely to need this article query again elsewhere, and its functionality isn't tied solely to posting a new comment. I would refactor to extract this into its own model for use again later